config/default.js

```diff
@@ -18,5 +18,14 @@ module.exports = {
 	s3 : {
 		access : '',
 		secret : ''
+	},
+	activitybot : {
+        debug : false,
+        ignored_channels: 'general,feedback,events,diagnostics',
+        target_channel: 'general',
+        threshold : {
+            message_count: 15,
+            seconds: 300
```

Comment:
Is it uncommon to see only 15 messages in 5 minutes?  I feel like this bot might be a little _eager_.   But I suppose that can be validated and tweaked as we go.

Reply:
Yeah, I originally had it set to 20, based loosely on the recent #design-dabblers outburst. Upping it back to 20.